apps/src/code-studio/components/AdvancedShareOptions.jsx

```diff
@@ -105,7 +104,6 @@ class AdvancedShareOptions extends React.Component {
     shareUrl: PropTypes.string.isRequired,
     allowExportExpo: PropTypes.bool.isRequired,
     exportApp: PropTypes.func,
-    librariesEnabled: PropTypes.bool,
```

Comment:
`librariesEnabled` should continue to exist. It's a levelbuilder flag.

Reply:
isn't it redunant now that libraries are always enabled? or am i misunderstanding what this property does?

Reply:
Libraries should only be enabled on certain levels. It should be enabled on the /projects/new page as well as certain CSP levels (where it's already enabled). But should be disabled in all others.

Reply:
Actually, that reminds me. We need to update the projects/new level to enable libraries

Reply:
ah, okay. so this logic needs to change so that if `librariesEnabled === false`, we hide libraries? 

to your second comment, i'm not sure what you mean by "update the project/new level" -- where does that happen?

Reply:
we chatted offline, so summarizing here -- each lab has it's own .level file for new projects of that lab type, so we just need to add `librariesEnabled: true` to the [applab new project level](https://github.com/code-dot-org/code-dot-org/blob/4b2aa4a4556a17e8976f46b8507889d947922646/dashboard/config/scripts/levels/New%20App%20Lab%20Project.level)